### ISSUE TITLE
Generating two different files with different output styles

### ISSUE THREAD

**Issue by rodrigodagostino** [oklai/koala#470]
Is there any way in which I can generate two different CSS files when outputting from a scss file with Koala? I am in need of a nested and a compressed version, in which the second file should have `.min` before `.css`.

Any help would be very much appreciated :)

**Comment by dougcassidy** [comment]
I cant find the instructions for this, anymore, so here is the bottom of my config.rb:

`output_style = :compressed






require 'fileutils'
require 'compass'
require 'compass/sass_compiler'

on_stylesheet_saved do |file|
  if file.match('.debug') == nil
    Compass.add_configuration({        
            :output_style  => :nested,
            :firecompass   => true,
			:line_comments => true
        },
        'debug' #ADDING A CONFIG REQUIRES A NAME
    )
	Compass.compiler.compile('style.scss', 'style.debug.css')                  #Compass.compiler is deprecated. Use Compass.sass_compiler instead.	 # 
	#Compass.sass_compiler.compile('style.scss', 'style.debug.css')	 #yabbut, it dont work. the deprecated one does...

  end #if
end #on


on_stylesheet_saved do |file|
  	$filename = "style.css"
  	$debugfilename = "style.debug.css"
	
  if File.exists?(file) && File.basename(file) == $filename
    FileUtils.mv(file, File.dirname(file) + "/../" + $filename)
  end
  if File.exists?(file) && File.basename(file) == $debugfilename
    FileUtils.mv(file, File.dirname(file) + "/../" + $debugfilename)
  end
end
`

**Comment by DTails** [comment]
I'm not really sure how to use that. Will that work for different filenames or just for style.scss?

**Comment by dougcassidy** [comment]
Yes, you'd just have to change the filenames.  You could probly put the filenames into variables, like in the "on_stylesheet_saved do" section.  But, I dont know ruby, I patched this stuff together, by stabbing in the dark.  

 As you can probly tell, my style.css is minified and style.debug.css has all the comments.  When I serve the stylesheet, I check whether the IP address is me, and give everyone else the minified.

**Comment by dougcassidy** [comment]
Yeah, actually, the "on_stylesheet_saved do"  moves the style.css and debug, out to the containing directory, you might not need that.  I'm in wordpress.  I have 
theme/
    css/
        style.scss //this is just an importer file for the partials
        config.rb
        partials/
            _these-are-all-underscored.scss  //so a bunch of css files dont get created
style.css
style.debug.css

**Comment by derekbtw** [comment]
This thread has been dead for a while now but I was wondering if anyone figured out a way to do this?

I want to output an extended version and a .min compressed version..

**Comment by zaygraveyard** [comment]
I think it can be done by using two build steps:

1. style.scss -> style.css (no minification)
2. style.css -> style.min.css (minification)

**Comment by dougcassidy** [comment]
My code, above, works great; I use it on tons of sites.
